src/ontology/mondo-ingest.Makefile

```diff
@@ -413,18 +413,20 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
-tmp/merged.db: tmp/merged.owl
+%.db: %.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
-	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	RUST_BACKTRACE=full semsql make tmp/$*.db -P config/prefixes.csv
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
-mondo-ingest.db: tmp/merged.db
+merged.db: tmp/merged.db
 	cp $< $@
-.PRECIOUS: mondo-ingest.db
+
+tmp/mondo-ingest.db: mondo-ingest.db
```

Comment:
## Add: `tmp/mondo-ingest.db`
This variation of the goal actually comes from `mondo-ingest.owl` and not from `tmp/merged.owl`. Putting it in  `src/ontology/tmp/` instead of `src/ontology/` (which is where `mondo-ingest.owl` is) is to be standard with where the other `.db` files are. It has no reason to be in `src/ontology/`. I'm not sure why `mondo-ingest.owl` is going there TBH.